model_compression_toolkit/quantizers_infrastructure/keras/quantizer_utils.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+import tensorflow as tf
+
+from model_compression_toolkit.quantizers_infrastructure.common.constants import EPS, MULTIPLIER_N_BITS
+
+
+def lut_kmeans_quantizer(tensor_data: tf.Tensor,
```

Comment:
if this function quantizes a tensor according to pre-defined cluster centers, then kmeans isn't relevant here.